spec/unit/network/http/pool_spec.rb

```diff
@@ -124,16 +124,14 @@ def create_connection(site)
       # This workaround can be removed once all the ruby versions we care about
       # have the patch from https://bugs.ruby-lang.org/issues/11958 applied.
       #
-      keepalive   = Socket::Option.bool(:INET, :SOCKET, :KEEPALIVE, true).data
-      nokeepalive = Socket::Option.bool(:INET, :SOCKET, :KEEPALIVE, false).data
-
       if Puppet::Util::Platform.windows?
-        keepalive   = keepalive[0]
-        nokeepalive = nokeepalive[0]
+        keepalive   = Socket::Option.bool(:INET, :SOCKET, :KEEPALIVE, true).data[0]
+        nokeepalive = Socket::Option.bool(:INET, :SOCKET, :KEEPALIVE, false).data[0]
+        expect(s.getsockopt(Socket::SOL_SOCKET, Socket::SO_KEEPALIVE).data).to eq(keepalive)
+        expect(s.getsockopt(Socket::SOL_SOCKET, Socket::SO_KEEPALIVE).data).to_not eq(nokeepalive)
+      else
+        expect(s.getsockopt(Socket::SOL_SOCKET, Socket::SO_KEEPALIVE).bool).to be true
```

Comment:
I'm used to seeing `be_truthy` or `eq(true)`. If you say `be` it will check that it's not nil. Does `be true` ignore the `true` and check that it's not nil? or does it do the same as the old rspec 2.x syntax `be_true`?